### PR TITLE
Feature/#132 private games

### PR DESCRIPTION
**Pull request by oconn**
### Problem

There is currently no support method for joining private games. 

### Solution

Add a password field to the join game modal for private games. This field will pass the required information to enroll users in private games and will only display for private games.

### CODE REVIEW

File: less/components/modals/modal.less [hunk 1]
```diff
@@ -12,9 +12,12 @@
     }
 
     .modal {
+        // We should look at what it would take
+        // to switch this over to using flex-box
+        // to center the content.
         width: 370px;
         position: fixed;
-        top: 50%;
+        top: 35%;
```
Comment (emilyseibert) [inline less/components/modals/modal.less]: why is this 35%

File: src/cljs/wombats_web_client/constants/urls.cljs [hunk 2]
```diff
@@ -6,10 +6,10 @@
                            :config-panel "config"})
 
 ;; Remote Dev
-(def base-api-url "//dev.api.wombats.io")
+#_(def base-api-url "//qa.api.wombats.io")
```
Comment (emilyseibert) [inline src/cljs/wombats_web_client/constants/urls.cljs]: noo :(

File: less/components/modals/modal.less (hunk 1 above)
Comment (oconn) [inline less/components/modals/modal.less]: the modal will be off the screen if its not

File: src/cljs/wombats_web_client/constants/urls.cljs (hunk 2 above)
Comment (oconn) [inline src/cljs/wombats_web_client/constants/urls.cljs]: sorry :(

File: src/cljs/wombats_web_client/constants/urls.cljs [hunk 3]
```diff
@@ -6,7 +6,7 @@
                            :config-panel "config"})
 
 ;; Remote Dev
-(def base-api-url "//dev.api.wombats.io")
+(def base-api-url "//qa.api.wombats.io")
```
Comment (dehli) [inline src/cljs/wombats_web_client/constants/urls.cljs]: oops, this should be `dev` on the develop branch.

Comment (emilyseibert) [inline src/cljs/wombats_web_client/constants/urls.cljs]: lol i should write better comments on prs

Comment (dehli) [inline src/cljs/wombats_web_client/constants/urls.cljs]: i think your comments are great! it's impossible to catch everything.

File: less/components/modals/modal.less (hunk 1 above)
Comment (dehli) [inline less/components/modals/modal.less]: I created #137 since it will eventually be nice to use flex-boxes (even if we don't go down that route yet).

Comment (oconn) [inline less/components/modals/modal.less]: Thanks! Merging for now, we'll punt on flex-box until post alpha

**Pull request merged**